kapp/Makefile

```diff
@@ -0,0 +1,23 @@
+PODTATO_HEAD_VERSION ?= v0.3.1
+
+.PHONY: install
+install:
+	mkdir kapp-bin/
+	curl -L https://carvel.dev/install.sh | K14SIO_INSTALL_BIN_DIR=kapp-bin bash
+	export PATH=$PWD/local-bin/:$PATH
+	kapp version
+
+.PHONY: uninstall
+uninstall:
+	kubectl delete ns podtato-kapp --ignore-not-found=true
+	rm -rf kapp-bin/
+
+.PHONY: deploy-app
+deploy-app:
+	kubectl create ns podtato-kapp || true
+	kustomize build ./app | kapp deploy -a podtato-head -n podtato-kapp --yes -f -
```

Comment:
This looks good!

I was wondering if different parts of podtato app are templated using different tools. Might be cool to show `kapp` plays well with all these tools.
It is illustrated here, but not obvious!